cleanlab/filter.py

```diff
@@ -444,9 +444,9 @@ class 0, 1, ..., K-1.
         label_issues_mask = find_predicted_neq_given(labels, pred_probs, multi_label=multi_label)
 
     if filter_by not in ["low_self_confidence", "low_normalized_margin"]:
-        # Remove label issues if given label == model prediction if issues haven't been removed yet
-        pred = pred_probs.argmax(axis=1)
-        label_issues_mask[pred == labels] = False
+        # Remove label issues if model prediction is close to given label
+        mask = _reduce_issues(pred_probs=pred_probs, labels=labels)
+        label_issues_mask[mask] = False
```

Comment:
```suggestion
        mask = _reduce_issues(pred_probs=pred_probs[cl_error_indices], labels=labels[cl_error_indices])
        label_issues_mask[cl_error_indices[mask]] = False
```

Reply:
On second thought. The variable `cl_error_indices` is not always defined at this point, so the mask needs to be applied to the full set of `pred_probs` and `labels`. 🤔